app/mailers/application_submission_mailer.rb

```diff
@@ -1,11 +1,12 @@
 class ApplicationSubmissionMailer < ApplicationMailer
-  def application_submitted_mail(to:, full_name:, provider_name:, course_name:)
+  def application_submitted_mail(_template_id, to:, full_name:, provider_name:, course_name:, amount:)
```

Comment:
`template_id` will be included once green light has been given that they can go live